frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -16,8 +16,9 @@ import { getLegend } from '../MapLegend/MapLegend'
 
 import './DownloadAnalysisLink.css'
 
-const useStaticCsvs: boolean = CONFIG.staticAssets.useStaticCsvs
-const staticCsvRootUrl: string = CONFIG.staticAssets.analysisResultsRootUrl
+const useStaticCsvs: boolean = CONFIG.staticAssets.csv.useStaticCsvs
+const staticCsvRootUrl: string = CONFIG.staticAssets.rootUrl
```

Comment:
why is this not under .csv with the others?

Reply:
Because it it its own root?

Reply:
Because I changed the configuration to have only a single root URL like you asked 😛

Reply:
Yeah I mean theoretically each type of asset could have its own bucket or CDN distro - just seemed to make sense to me to keep em all in the same public encompass data bucket